lnwallet/fee_estimator_test.go

```diff
@@ -144,3 +160,71 @@ func TestSparseConfFeeSource(t *testing.T) {
 		t.Fatalf("expected ParseResponse to fail")
 	}
 }
+
+func TestWebAPIFeeEstimator(t *testing.T) {
```

Comment:
Missing `godoc` comment for the test.

Reply:
Not clear that this test covers the case of falling back to a lower estimate if the target num blocks conf isn't found in the map. The `getCachedFee` method may need to be test directly in order to achieve this level of coverage.

Reply:
fixed & fixed